api/internal/target/kusttarget.go

```diff
@@ -425,7 +425,14 @@ func (kt *KustTarget) accumulateResources(
 			}
 			ldr, err := kt.ldr.New(path)
 			if err != nil {
-				if kusterr.IsMalformedYAMLError(errF) { // Some error occurred while tyring to decode YAML file
+				// If accumulateFile found malformed YAML and there was a failure
+				// loading the resource as a base, then the resource is likely a
+				// file. The loader failure message is unnecessary, and could be
+				// confusing. Report only the file load error.
+				//
+				// However, a loader timeout implies there is a git repo at the
+				// path. In that case, both errors could be important.
+				if kusterr.IsMalformedYAMLError(errF) && !utils.IsErrTimeout(err) {
```

Comment:
Is there a situation where `kusterr.IsMalformedYAMLError()` will result `true` and `utils.IsErrTimeout()` will also result `true`?

Reply:
Yes, the scenario is described in #5540.

Reply:
Makes sense -- feel free to mark this as resolved.

Reply:
Thanks!